### PR TITLE
homework_week1

### PR DESCRIPTION
**Pull request by sumanghimire79**

### CODE REVIEW

File: week1/homework/Program.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  Console.WriteLine($"Reversed input value: {reversed}");
+  return ($"Reversed input value: {reversed}");
+});
+
+//2. String/Math
+// Return the number (count) of vowels in the given string. Consider a, e, i, o, u as vowels.
+app.MapGet("/2", () =>
+{
+  string input = "IntellectualizationOOYa";
+  int vowelCount = GetVowelCount(input.ToLower()); //TODO: Implement GetVowelCount
```
Comment (calindurnea) [inline week1/homework/Program.cs:32]: good consideration to add `ToLower` 💯

File: week1/homework/Program.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+//Given an array of the numbers return an array with two elements where first element represents sum of all negative numbers and second element represents multiplication of all positive numbers;
+
+app.MapGet("/3", () =>
+{
+  int[] arr = new[] { 271, -3, 1, 14, -100, 13, 2, 1, -8, -59, -1852, 41, 5 };
+  int[] result = GetResult(arr); //TODO: Implement GetResult
+  int[] GetResult(int[] array)
+  {
+    int countP = 0;
+    int countN = 1;
```
Comment (calindurnea) [inline week1/homework/Program.cs]: it is generally better to gibe full names for the variables, it makes the code easier to understand.

Here it is a bit confusing:
1. I see the `countP` variable and I assume the `P` is for `positive` based on the task
2. In the implementation below you are doing `if (item < 0) countP += item;` which is the use case if the numbers are `negative`

File: week1/homework/Program.cs [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      if (item > 0) countN *= item;
+    }
+    return new[] { countP, countN };
+  }
+
+  Console.WriteLine($"Sum of negative numbers: {result[0]}. Multiplication of positive numbers: {result[1]}");
+  return ($"Sum of negative numbers: {result[0]}. Multiplication of positive numbers: {result[1]}");
+});
+
+
```
Comment (calindurnea) [inline week1/homework/Program.cs:75]: I think you are missing task `#4` 🤔

**Review by calindurnea — APPROVED** [review]
Nice job 🥇 
I left a few comments, please reach out if you have any questions

File: week1/homework/Program.cs (hunk 1 above)
Comment (sumanghimire79) [reply week1/homework/Program.cs:32]: thank you

File: week1/homework/Program.cs (hunk 2 above)
Comment (sumanghimire79) [reply week1/homework/Program.cs]: yes i corrected it . thank you

File: week1/homework/Program.cs (hunk 3 above)
Comment (sumanghimire79) [reply week1/homework/Program.cs:75]: i have added this one. thank you

**Pull request merged**